Makefile

```diff
@@ -90,14 +83,6 @@ dist: build-cross
 clean:
 	@rm -rf '$(BINDIR)' ./_dist
 
-.PHONY: docker-build
-docker-build:
-	docker build -f Dockerfile -t $(ITER8_IMG) .
-
-.PHONY: docker-push
-docker-push:
-	docker push $(ITER8_IMG)
-
```

Comment:
Why removing these commands? I know we are building our image with GH actions. However, as part of the development process I have often built my own image for testing. I find these make commands useful for this purpose.  It is true that the need has lessened significantly starting with v0.8 but I have done it several times since.

Reply:
In this case, we will keep it.